### PR TITLE
Add Pokémon

### PR DESCRIPTION
**Pull request by tobiasbueschel**
**[https://github.com/tobiasbueschel/awesome-pokemon]**

**[Awesome-pokemon is a curated list of awesome Pokemon & Pokemon Go resources, tools and more. It should be included because a lot of Pokemon related development is currently happening in the community see e.g. [/r/pokemondev](https://www.reddit.com/r/pokemongodev) and this list helps to collect all helpful resources.]**

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a dot, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Nice!

Some nitpick:

- `Pokemon` => `Pokémon` (everywhere)
- `PokéMon` => `Pokémon`
- `pokemon` => `Pokémon`
- `pokemon go` => `Pokémon Go`
- `Inspired by the awesome list thing.` is not really needed. You already have the badge.
- > starting each link description with an uppercase character and **ending it with a dot**
- `This tool shows you` => `Shows you`
- "PokemonHook" is missing description
- `Node` => `Node.js`
- `game/api` => `game/API`

**Comment by sindresorhus** [comment]
Oh, and I made a logo for you. Pick either of these: (Or none if you don't like it)

awesome-pokemon 1
awesome-pokemon

File: readme.md [hunk 1]
```diff
@@ -436,6 +436,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Bots](https://github.com/hackerkid/bots) - Building bots.
 - [Site Reliability Engineering](https://github.com/dastergon/awesome-sre)
 - [Empathy in Engineering](https://github.com/KimberlyMunoz/empathy-in-engineering) - Building and promoting more compassionate engineering cultures.
+- [Pokemon](https://github.com/tobiasbueschel/awesome-pokemon) - A curated list of awesome Pokemon & Pokemon Go resources, tools and more.
```
Comment (sindresorhus) [inline readme.md]: `Pokémon`

Comment (sindresorhus) [inline readme.md]: Description:

> Pokémon and Pokémon GO resources and tools.

**Comment by tobiasbueschel** [comment]
Thanks for the feedback @sindresorhus! Some of these things must have slipped under my radar - sorry for the inconvenience! I made the respective changes in [awesome-pokemon](https://github.com/tobiasbueschel/awesome-pokemon) and also updated the PR to include the correct spelling & description! 

I love the new logo!! Thank you very much for that :tada: :tada:

Oh and for link names such as **[pokemon-go-mitm-node]** I did not include the correct Pokémon GO spelling as these are the names developers have given to their repos / projects! Hope that's okay :)

Let me know in case I missed something else.

Best wishes,
Tobias

**Comment by aleksandar-todorovic** [comment]
@tobiasbueschel Could you squash the conflicts?

**Comment by sindresorhus** [comment]
Just fix the merge conflict and this is good to go :)

**Comment by tobiasbueschel** [comment]
it should be good to go now :) Thanks for the tips and patience @sindresorhus @aleksandar-todorovic

**Pull request merged**